openslides/core/config_variables.py

```diff
@@ -130,6 +130,19 @@ def get_config_variables():
         group='General',
         subgroup='Export')
 
+    yield ConfigVariable(
+        name='general_export_pdf_fontsize',
+        default_value='10',
+        input_type='choice',
+        label='Standard font size in PDF',
+        choices=(
+            {'value': '10', 'display_name': '10'},
```

Comment:
It's just the _standard_ font size for text which sould be between 10 and 12pt. More is not useful because font size of h3 is still 12pt. And smaller than 10 is not readable for text; also because line numbers always 2 pt smaller (10pt text -> 8pt linenumbers). 
So, won't fix.